Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:13.12.0-alpine
```

Comment:
We should use `node:bullseye-slim` or `node:alpine` for this use case -- rather than pinning to a specific node image version. These are recommended ways to deploy standard, small node docker containers!

Reply:
Agreed, making the change now.